firebase-installations/src/androidTest/java/com/google/firebase/installations/FisAndroidTestConstants.java

```diff
@@ -59,6 +61,20 @@
           .setResponseCode(ResponseCode.OK)
           .build();
 
+  public static final InstallationResponse TEST_INSTALLATION_RESPONSE_WITH_IID =
+      InstallationResponse.builder()
+          .setUri("/projects/" + TEST_PROJECT_ID + "/installations/" + TEST_INSTANCE_ID_1)
```

Comment:
you could use a constant for this as well:
FIS_URI = "/projects/%s/installations/%s";

but not important.